Library/AppEnvironmentTests.swift

```diff
@@ -81,7 +93,9 @@ final class AppEnvironmentTests: XCTestCase {
     AppEnvironment.popEnvironment()
   }
 
-  func testFromStorage_WithNothingStored() {
+  func testFromStorage_WithNothingStored_featureUseKeychainEnabledIsFalse() {
+    self.setFeatureUseKeychainEnabled(false)
```

Comment:
So now we have tests running in three different states:

`AppEnvironmentTests` running in the un-hosted test environment, with the feature flag OFF. This lets us run the original `AppEnvironmentTests` without breaking all of them.
`AppEnvironmentTests` running in the hosted test environment, with the feature flag OFF. This simulates the app with the feature flag off.
`AppEnvironmentTests+OAuthInKeychain` running in the hosted test environment, with the feature flag ON. This simulates the app with the feature flag on.